config/routes.rb

```diff
@@ -33,7 +33,7 @@
   resources :services do
     resources :change_requests, only: :create
     resources :notes, only: :create
-    resources :feedbacks, only: %i[create index]
```

Comment:
Looks like you accidentally reverted the feedback routes here. Could you restore this line?